lua/packer/display.lua

```diff
@@ -571,6 +576,20 @@ local display_mt = {
     end
   end,
 
+  open_log = function(self)
+    if not self:valid_display() then return end
+    if next(self.items) == nil then
+      log.info('Operations are still running; plugin info is not ready yet')
```

Comment:
Should showing the log be tied to plugin operations running?

Reply:
If the user has not configured the buffer to be reloaded when the file changes outside of the editor, the logs will be static. (`:h autoread`, `:h checktime`) So, I thought it best to make the command available once packer has completed all the jobs.